ssr-testing/next.config.js

```diff
@@ -0,0 +1,19 @@
+const path = require('path');
+const withTM = require('next-transpile-modules')([]);
+
+module.exports = withTM({
+  webpack(config) {
+    // https://github.com/josephluck/next-typescript-monorepo/blob/master/blog/next.config.js
+    const babelRule = config.module.rules.find((rule) =>
+      rule.use && Array.isArray(rule.use)
+        ? rule.use.find((u) => u.loader === 'next-babel-loader')
+        : rule.use.loader === 'next-babel-loader'
+    );
+
+    if (babelRule) {
+      babelRule.include.push(path.resolve('../'));
+    }
+
+    return config;
+  },
+});
```

Comment:
This took me ages to figure out, it works but I don't really get how…

Reply:
well if it works, I guess that's all we need haha. It's just an internal thing anyway 🤷‍♀️